frontend/src/app/main/data/common.cljs

```diff
@@ -174,6 +174,12 @@
              (rx/tap on-success)
              (rx/catch on-error))))))
 
+(def change-role-msg
+  {:viewer "dashboard.permissions-change.viewer"
+   :editor "dashboard.permissions-change.editor"
+   :admin  "dashboard.permissions-change.admin"
+   :owner  "dashboard.permissions-change.owner"})
```

Comment:
I think instead of defining plain map, it should be defined as function, because you need to use the `tr` function statically visible for the `translations` script correctly collect used and unused translation strings.

With this approach, the resolution for `tr` some lines below is dynamic and can't be statically analized by external scripts

Reply:
fixed